test/Sentry.Tests/Internals/ProcessInfoTests.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Threading.Tasks;
+using Sentry.Internal;
+using Sentry.Testing;
+using Xunit;
+
+namespace Sentry.Tests.Internals
+{
+    public class ProcessInfoTests
```

Comment:
Could you please add a test that asserts a new instance of ProcessInfo starts with a `StartupTime` that's not in the future and it's less than `UtcNow` by up to a few seconds?